lib/types.nix

```diff
@@ -107,22 +107,80 @@ rec {
       merge = mergeEqualOption;
     };
 
+### INTEGERS
```

Comment:
nit: `### Integers` , do the same below for other comments. Indent this comment at the same level as the code.